### ISSUE TITLE
Scale together

### ISSUE THREAD

**Issue by tieugene** [tieugene/iosc.py#177]
Scale joined one-type signals together

**Comment by michDaven** [comment]
Евгений, я правильно понял, что речь идёт о том, в каком масштабе (вертикальном) отображать итоговый сигнал если сигнал 2x объединяется с сигналом 3x?

**Comment by tieugene** [comment]
> Евгений, я правильно понял, что речь идёт о том, в каком масштабе (вертикальном) отображать итоговый сигнал если сигнал 2x объединяется с сигналом 3x?

Нет. Масштабирование по вертикали - работает масштаб "ленты"-получатель (хоть это и не указано в ТЗ, я сам придумал).
Речь идет о том к какому маштабу приводить объединение сигналов одного... мнэ... типа, что ли. Например токи разных фаз.
Естественно с учетом того, как определять какие сигналы можно масштабирвоать вместе, что делать с остальными сигналами в этой же "ленте", как пересчитывать при добавление или удалении сигнала того же или другого типа и т.д. и т.п.

**Comment by michDaven** [comment]
Укажу тут, что сейчас механизм масштабирования описан в п. 5.2 "Масштабирование сигналов при их наложении в одном окне (ленте)"
